kahuna/public/js/preview/image.js

```diff
@@ -64,6 +64,8 @@ image.controller('uiPreviewImageCtrl', [
           ctrl.imageAsArray = [newImage];
       });
 
+      ctrl.showSendToPhotoSales = () => $window._clientConfig.showSendToPhotoSales;
+
```

Comment:
Does this need to be a function call here or can it be accessed directly as a variable? (changing to variable call would impact image.html)